xarray/tests/duckarrays/test_sparse.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+
+pytest.importorskip("hypothesis")
+
+from xarray import DataArray, Dataset, Variable
+
+from .. import assert_allclose
+from . import base
+from .base import strategies
+
+sparse = pytest.importorskip("sparse")
+
+pytestmark = [
+    pytest.mark.skip(
+        reason=(
+            "timing issues due to the JIT compiler of numba"
+            " and precision differences between sparse and numpy / bottleneck"
+        )
```

Comment:
This will unconditionally skip all tests in this file, which seems like it might be excessive?

If the timing problem is Hypothesis raising `Flaky` because you *sometimes* exceed the default deadline, you could instead decorate the relevant test methods with `@hypothesis.settings(deadline=None)` to disable it.

For the precision differences, I'd try using 

```python
pytest.mark.xfail(
    reason="precision differences between sparse and numpy / bottleneck",
    raises=AssertionError, strict=False
)
```

or look for a subset of inputs / relative tolerance where they do agree and just check that.